yarn.lock

```diff
@@ -6236,17 +6236,6 @@ chainlink@0.7.5:
     truffle-contract "^4.0.31"
     web3 "^1.2.0"
 
-chainlink@0.7.8:
```

Comment:
Let's make either a blocker or a release story so that we dont forget to publish the new package version

Reply:
the 0.7.10 version is already published. I just updated the dependency in `integration-scripts` to use the latest version.

Reply:
Sweet, thanks!